### ISSUE TITLE
Cannot be used with gtk 0.15

### ISSUE THREAD

**Issue by ArekPiekarz** [gtk-rs/gtk-test#51]
gtk-test 0.14 does not work with gtk 0.15. When trying to use them in one project, I get the following errors:
```
$ cargo build
    Updating crates.io index
error: failed to select a version for `gtk-sys`.
    ... required by package `gtk v0.14.0`
    ... which satisfies dependency `gtk = "^0.14"` of package `gtk-test v0.14.0`
    ... which satisfies dependency `gtk-test = "^0.14.0"` of package `example-app v0.1.0 (/home/user/dev/project/example-app)`
versions that meet the requirements `^0.14.0` are: 0.14.0

the package `gtk-sys` links to the native library `gtk-3`, but it conflicts with a previous package which links to `gtk-3` as well:
package `gtk-sys v0.15.1`
    ... which satisfies dependency `ffi = "^0.15.1"` of package `gtk v0.15.2`
    ... which satisfies dependency `gtk = "^0.15.2"` of package `example-app v0.1.0 (/home/user/dev/project/example-app)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the links ='gtk-sys' value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

failed to select a version for `gtk-sys` which could resolve this conflict
```

**Comment by GuillaumeGomez** [comment]
It hasn't been updated yet.

**Comment by GuillaumeGomez** [comment]
Just published the 0.15.0 version. Enjoy!
